src/packet/ControlPacket.php

```diff
@@ -11,19 +11,13 @@
 
 abstract class ControlPacket {
 
-    protected $command;
+//    protected $command;
```

Comment:
Can you please remove that? I think its not necessary anymore, as you figured out :+1: